functions/functions.inc

```diff
@@ -25,6 +23,52 @@ $MODULESDIR     = $BASEDIR."/modules";
 // Set a Date stamp for use anywhere
 $DS             = uniqid('', true);
 
+
+function getVersion()
```

Comment:
Hm. I think I'm OK with this for now, but I dislike having a requirement that `git` exist on the target server. The way we're deploying now, it's a safe assumption, but if we were to move to a very different model (say, docker), it might not be.

Reply:
Understood. The idea is that a Release version without git would have "build" set properly in the verison.ini file. 

I am toying with writing some sort of a packager that can be run to generate a nice .zip or .tgz file that is ready for deployment.  Does all the composer stuff and updates the "build" property etc... But that may not be worth it at this point.